javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,8 @@
+/* exported countdown */
+function countdown(number) {
+  var numbers = [];
+  for (var i = number; i > -1; i--) {
```

Comment:
Both methods work here, but this would be a great example of a place to use a while loop! Since we don't know how big `number` will be, it makes sense to use a while loop

Reply:
That makes sense! I suppose I could have done while(i !==0) and decrement within the while loop. Thanks for the helpful feedback!